tests/checks_test.js

```diff
@@ -74,7 +74,7 @@ test.serial.before(
             ...payload,
             to: {
               company: "Lob (old)",
```

Comment:
Noting that these descriptions haven't changed even if the address has changed to the new Lob location.